gslib/commands/lifecycle.py

```diff
@@ -137,6 +138,45 @@ class LifecycleCommand(Command):
       },
   )
 
+  gcloud_storage_map = GcloudStorageMap(
+      gcloud_command={
+          'get':
+              GcloudStorageMap(
+                  gcloud_command=[
+                      'alpha', 'storage', 'buckets', 'describe',
+                      '--format=json(lifecycle)'
+                  ],
+                  flag_map={},
+              ),
+      },
+      flag_map={},
+  )
+
+  def get_gcloud_storage_args(self):
+    if self.args[0] == 'set':
+      gcloud_storage_map = GcloudStorageMap(
+          gcloud_command={
+              'set':
+                  GcloudStorageMap(
+                      gcloud_command=[
+                          'alpha',
+                          'storage',
+                          'buckets',
+                          'update',
+                          '--lifecycle-file={}'.format(self.args[1]),
+                      ] + self.args[2:],  # Adds target bucket URLs.
+                      flag_map={},
+                  ),
+          },
+          flag_map={},
+      )
+      # Don't trigger unneeded translation now that complete command is above.
+      self.args = self.args[:1]
+    else:
+      gcloud_storage_map = LifecycleCommand.gcloud_storage_map
```

Comment:
Nit: You can probably add the map directly here, will be consistent with the set section above.

Reply:
Done